src/modules/customer/HappyUserComponents/BodyGroups.js

```diff
@@ -0,0 +1,7 @@
+import React, { Component } from 'react';
```

Comment:
This is also a reusable component, to be used in many modules. Moreover, it seems to be a container, that will have its own components. So move it to containers.
I think you should rename it. As far as i remember, it will be kind of navigator or map of the collections of health problems / medicines, etc. So lets think up a better name for this container. It will have search, filter, expand lists functionalities. It will have many components. The main functionality of it is to enable user surf / search all the products.

ProductsCollection seems a better name. Maybe you will think up a better one.